apps/ledger-live-desktop/src/renderer/families/evm/EditTransaction/steps/StepFees.tsx

```diff
@@ -0,0 +1,158 @@
+import { getMainAccount } from "@ledgerhq/live-common/account/index";
+import { getFormattedFeeFields } from "@ledgerhq/live-common/families/evm/editTransaction/index";
+import React, { Fragment, memo } from "react";
+import { Trans } from "react-i18next";
+import { useSelector } from "react-redux";
+import Alert from "~/renderer/components/Alert";
+import Box from "~/renderer/components/Box";
+import Button from "~/renderer/components/Button";
+import BuyButton from "~/renderer/components/BuyButton";
+import CurrencyDownStatusAlert from "~/renderer/components/CurrencyDownStatusAlert";
+import logger from "~/renderer/logger";
+import SendAmountFields, { SendAmountFieldsProps } from "~/renderer/modals/Send/SendAmountFields";
+import { localeSelector } from "~/renderer/reducers/settings";
+import { TransactionErrorBanner } from "../components/TransactionErrorBanner";
+import { StepProps } from "../types";
+
+const StepFees = ({
+  account,
+  parentAccount,
+  transaction,
+  transactionToUpdate,
+  status,
+  bridgePending,
+  t,
+  onChangeTransaction,
+  updateTransaction,
+}: StepProps) => {
+  const mainAccount = account ? getMainAccount(account, parentAccount) : null;
```

Comment:
Yes. Actually the typing of `account` under `StepProps` is wrong
https://github.com/LedgerHQ/ledger-live/blob/1babf55a1232e89711ac69575805496e8615f58b/apps/ledger-live-desktop/src/renderer/families/evm/EditTransaction/types.ts#L12

`account` is always defined at this stage since it's existence is already tested in `Body`
https://github.com/LedgerHQ/ledger-live/blob/d2145d48d0ccf502bd905d181bcbcd8218c4e712/apps/ledger-live-desktop/src/renderer/families/evm/EditTransaction/Body.tsx#L179

Will fix the typing and remove the unnecessary tests

Reply:
63c770ea2583f91d8cc562a3e703e498c480f0de